source/developers/website.markdown

```diff
@@ -94,7 +94,24 @@ When you're writing code that is to be executed on the terminal, prefix it with
 
 For the [configuration templating](/topics/templating/) is [Jinja](http://jinja.pocoo.org/) used.
 
-If you are using templates then those parts needs to be [escaped](http://stackoverflow.com/a/24102537). Otherwise they will be rendered and appear blank on the website.
+If you are using templates then those parts needs to be [escaped](http://stackoverflow.com/a/24102537). Otherwise they will be rendered and appear blank on the website:
+
+```yaml
+# Example configuration.yml entry.
+automation:
+  - alias: "Example notification"
+    trigger:
+      - platform: state
+        entity_id: sensor.example
+    action:
+      - service: notify.notify
+        data:
+          title: "Test"
+          message: >-
+            {% raw %}{% if is_state("sensor.example", "test") %}
```

Comment:
The issue is that this way the template will not be rendered but you still can't see where `{% raw %}` and `{% endraw %}` are placed without looking at the source of the page. We tried to solve already but there was no smart way to do it. We decided not to create an image as the provided link show a couple of examples.